doc/langref.html.in

```diff
@@ -4660,13 +4660,35 @@ test "for basics" {
     try expect(sum == 20);
 
     // To access the index of iteration, specify a second condition as well
-    // as a second capture value.
+    // as a second capture value using the range syntax.
```

Comment:
// To access the index of iteration, specify a second condition using range syntax as well
    // as a second capture value.

Reply:
Done